.github/workflows/compile.yml

```diff
@@ -17,7 +17,7 @@ jobs:
       - name: Set up Go
         uses: actions/setup-go@v3
         with:
-          go-version: 1.19
+          go-version: 1.23
```

Comment:
imo this should be one of the very first commits in the PR as CI will be broken until then (not important, can be ignored, I suspect making the code "1.23" ready and then switching it to 1.23 for good, and ensuring everthing build at all steps will be more work than it is worth it)

Reply:
Done